client/keys/show.go

```diff
@@ -30,16 +39,20 @@ func getKey(name string) (keys.Info, error) {
 // CMD
 
 func runShowCmd(cmd *cobra.Command, args []string) error {
-	if len(args) != 1 || len(args[0]) == 0 {
-		return errors.New("You must provide a name for the key")
+	info, err := getKey(args[0])
```

Comment:
We should still check the length of `args` and print a helpful error if it isn't equal to 1.